src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/PackageWrapper.cs

```diff
@@ -52,9 +52,7 @@ public static PackageWrapper GetWrapperFromPackage(Package package)
             string path;
             try
             {
-                path = IsPackageDotInstallationPathAvailable.Value
-                    ? package.InstalledPath
-                    : package.InstalledLocation.Path;
+                path = IsPackageDotInstallationPathAvailable.Value ? GetInstalledPath(package) : package.InstalledLocation.Path;
```

Comment:
Doesn't it work the same as before? `package.InstalledPath` was moved into the method. This shouldn't change behavior.

Reply:
I retested when I didn't have a seperate method, but as soon as the application reached this method it would throw an MethodNotFoundException (which makes sense if it tries to resolve a method pointer which not exists).